db/schema.rb

```diff
@@ -1,4 +1,3 @@
-# encoding: UTF-8
```

Comment:
Can you revert all these changes? db/schema.rb shouldn't be updated in a styling update.

Reply:
Agh! Sorry.

Reply:
We shouldn't delete this file, only undo the changes to it made in e6110236761222ec13c4cf14b5bbb94540872cbf.